src/components/NewCommentForm/NewCommentForm.jsx

```diff
@@ -1,39 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
 import './NewCommentForm.scss';
+import { addComment } from '../../api/comments';
+
+export const NewCommentForm = ({ postId, showComments }) => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [body, setBody] = useState('');
 
-export const NewCommentForm = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+  const sendComment = (event) => {
+    event.preventDefault();
+    addComment(postId, name, email, body)
+      .then(showComments);
+    setName('');
+    setEmail('');
+    setBody('');
+  };
+
+  return (
+    <form
+      className="NewCommentForm"
+      onSubmit={event => sendComment(event)}
     >
-      Add a comment
-    </button>
-  </form>
-);
+      <div className="form-field">
+        <input
+          type="text"
+          name="name"
+          placeholder="Your name"
+          className="NewCommentForm__input"
+          value={name}
+          onChange={event => setName(event.target.value)}
```

Comment:
```suggestion
          onChange={({ target }) => setCommentPart(target.name, target.value)}
```
create `setCommentPart` method

```javascript
const setCommentPart  = (name, value) => setComment(prevComment => ({
  ...prevComment,
  [name]: value,
}))
```
and pass it to every input